client/home.js

```diff
@@ -40,7 +40,6 @@ const logOutBtn = document.querySelector('#logOut')
 
 logOutBtn.addEventListener('click', async e => {
 	let response = await fetch('/logout')
```

Comment:
If response isn't going to be redefined somewhere else, then using `const` is preferred.

Same goes for `resJson` on the next tline

Reply:
Done